release-1.8/release_notes_draft.md

```diff
@@ -102,6 +105,11 @@ kind.
  Users that have any code that depends on mutable selectors may continue to use 
  the apps/v1beta1 API for this release, but they should begin migration to code 
  that does depend on mutable selectors.
+ - Extended Resources outside the `*kubernetes.io/` namespace are treated as
```

Comment:
outside of `kubernetes.io` and non namespaced resources

Reply:
Not sure if that will make sense in this sentence. Extended Resources by definition are outside the kubernetes.io domain. How about rewriting as:

"Extended Resources are fully-qualified resource names outside of the `kubernetes.io` domain. Extended Resource quantities must be integers. Extended Resources cannot be overcommitted..."

Reply:
Even i tried a bit :-)

ERs are fully-qualified resource names outside of the `kubernetes.io` domain. ER values must be integers and cannot be overcommitted.

(Using ER because we have introduced an abbreviation in the first sentence)

Reply:
+1 for either option.